common/lib/xmodule/xmodule/modulestore/tests/test_mongo.py

```diff
@@ -1,47 +1,48 @@
 """
 Unit tests for the Mongo modulestore
 """
-# pylint: disable=protected-access
+
 # pylint: disable=no-name-in-module
 # pylint: disable=bad-continuation
-from django.test import TestCase
-# pylint: enable=E0611
-from path import Path as path
-import pymongo
-import pytest
+from __future__ import absolute_import
+
 import logging
 import shutil
+from datetime import datetime
 from tempfile import mkdtemp
 from uuid import uuid4
-from datetime import datetime
-from pytz import UTC
+
+import pymongo
+import pytest
+import six
+# pylint: disable=protected-access
+from django.test import TestCase
 from mock import patch
+from opaque_keys.edx.keys import CourseKey, UsageKey
+from opaque_keys.edx.locator import AssetLocator, BlockUsageLocator, CourseLocator, LibraryLocator
+# pylint: enable=E0611
```

Comment:
This is supposed to stay right after the `TestCase` import, it restores checking for the `no-name-in-module` warning.